Neos.Flow/Classes/Error/ProductionExceptionHandler.php

```diff
@@ -39,7 +40,15 @@ protected function echoExceptionWeb($exception)
         try {
             if ($this->useCustomErrorView()) {
                 try {
-                    echo $this->buildView($exception, $this->renderingOptions)->render();
+                    $stream = $this->buildView($exception, $this->renderingOptions)->render();
+                    if ($stream instanceof ResponseInterface) {
+                        /**
+                         * The http status code will already be sent, and we are only currently interested in the content stream
+                         * Thus, we unwrap the repose here:
+                         */
+                        $stream = $stream->getBody();
+                    }
```

Comment:
@Sebobo we just talked about that the `FusionExceptionView`'s headers are removed by this logic.
What would be the desired logic?

We generate the `header` from the exceptions status code currently.

Reply:
Yeah we can add the headers but which status code should we use? The one inferred from the FlowException / 500 or the one from the view?

Maybe: use from the view if it’s NOT 200 and else fallback to the current logic?

Reply:
See https://github.com/neos/neos-development-collection/issues/4964 and can be fixed as follow up. Behaviour did not change yet, but we have the possibility to fix it ;)